src/components/Button/Loader/styles.js

```diff
@@ -0,0 +1,69 @@
+import styled, { keyframes } from "styled-components";
+
+import { getThemeValue } from "../../../utils";
+
+const dotColors = {
+  white: getThemeValue("white", "base"),
+  primary: getThemeValue("primary", "base"),
+  dark: getThemeValue("darkFill")
+};
+
+const colorVariants = {
+  standard: dotColors.white,
+  standardDisabled: dotColors.white,
+  special: dotColors.white,
+  specialDisabled: dotColors.white,
+  outline: dotColors.primary,
+  outlineDisabled: dotColors.primary,
+  transparent: dotColors.primary,
+  transparentDisabled: dotColors.primary,
+  outlineGray: dotColors.dark,
+  outlineGrayDisabled: dotColors.primary
+};
+
+const wave = keyframes`
+  0%, 60%, 100% {
+    transform: initial;
+  }
+
+  30% {
+    transform: translateY(-10px);
+  }
+`;
+
+export const Dot = styled.span`
+  display: inline-block;
+  width: 6px;
+  height: 6px;
+  border-radius: 50%;
+  background: ${({ variant }) => colorVariants[variant]};
+  animation: ${wave} 0.8s linear infinite;
+
+  &:nth-child(2) {
+    animation-delay: -0.1s;
+    margin: 0 8px;
```

Comment:
Do we have arora constant for 8px? If yes we should use it here

Reply:
Updated.